src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4868,12 +4768,12 @@ private static NamespaceOrTypeSymbol ToLanguageSpecific(INamespaceOrTypeSymbol c
                 return null;
             }
 
-            var result = container as NamespaceOrTypeSymbol;
-            if ((object)result == null)
+            if (!(container is NamespaceOrTypeSymbol namespaceOrTypeSymbol))
```

Comment:
On second thought, don't bother. This is a recurring pattern in other methods anyways.

---
In reply to: [121507519](https://github.com/dotnet/roslyn/pull/19941#discussion_r121507519) [](ancestors = 121507519)